### DISCUSSION TITLE
Debe ser el módulo de avatar un módulo de Mistica?

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1731] [Proposals]
### Goal

Saber si tiene sentido que el módulo de avatar sea Mistica o un extended

### Context

Tras la migración de páginas nativas al CMS, le toca el turno a la página de perfil, y eso significa que elementos que antes eran nativos, deban ser creados para webapp. Este es el caso del componente avatar+nombre

### Impact

Ese módulo es necesario en todas las Obs aunque con configuraciones distintas, debe ser un módulo con bastante configuración.

### Discussion Questions

-¿Tiene sentido que sea Mistica?
-¿Tiene sentido que sea un extended?

### Do you have a visual proposal?

https://www.figma.com/file/fmax7C03DUocQPrFBSNywj/User-profile-data?type=design&node-id=2001-6729&mode=design

**Comment by aweell** [comment]
Buenas @MontseTelefonica,

1. Plantear el modulo entero a priori como parte de Mística no acaba de encajar mucho, al final es una construcción de contenido que puede ser perfectamente atómica y que parece servir un propósito en el producto muy concreto (así como sus configuraciones por marca).
3. Lo que nos parece interesante evolucionar es la configuración del avatar para que puedan añadirse ese tipo de "satélites"

Con respecto al punto 2 ya existe una issue para cubrir esa necesidad: https://github.com/Telefonica/mistica-design/issues/1406

Una vez actualizado el avatar soportando esa configuración podría reutilizarse en la definición de este módulo.

**Reply by MontseTelefonica** [reply]
Perfecto! Continuamos como extended! FYI @sandrafernandezsimon

**Reply by MontseTelefonica** [reply]
Hola! Para cuando se cubrirá el caso de los satélites? es urgente para el replancement de O2ES

**Reply by lauraocana** [reply]
@MontseTelefonica
